llvm/test/CodeGen/AMDGPU/bf16-math.ll

```diff
@@ -25,4 +25,27 @@ define amdgpu_ps void @llvm_log2_bf16_s(ptr addrspace(1) %out, bfloat inreg %src
   ret void
 }
 
+define amdgpu_ps void @llvm_exp2_bf16_v(ptr addrspace(1) %out, bfloat %src) {
+; GCN-LABEL: llvm_exp2_bf16_v:
+; GCN:       ; %bb.0:
+; GCN-NEXT:    v_exp_bf16_e32 v2, v2
+; GCN-NEXT:    global_store_b16 v[0:1], v2, off
+; GCN-NEXT:    s_endpgm
+  %exp = call bfloat @llvm.exp2.bf16(bfloat %src)
```

Comment:
If it's OK to directly select the generic intrinsic, why add the amdgcn one?

Reply:
I don't follow

Reply:
The target intrinsic `int_amdgcn_exp2` was added a long time ago (along with the pattern `AMDGPUexpf16`). We didn't explicitly define a bf16 version of it. On the other hand, the generic intrinsic support was only added about a month ago.